common/taskana-common-test/pom.xml

```diff
@@ -51,6 +51,11 @@
       <groupId>com.fasterxml.jackson.datatype</groupId>
       <artifactId>jackson-datatype-jsr310</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>${version.javax.servlet}</version>
+    </dependency>
```

Comment:
I know that this is just a new test dependency. But why do we need it for the new spring version?
Do you have a reference by any chance?

Reply:
As far as I can see it's already included within spring: https://docs.spring.io/spring-boot/docs/2.6.3/reference/html/dependency-versions.html

Reply:
We already checked it and it's also in the effective pom. So it's delivered with spring. However, somehow it isn't recognized as a transitive dependency that's why we had to include it 'by-hand'.